lib/main_common.pm

```diff
@@ -1543,6 +1543,8 @@ sub load_nfv_master_tests {
     load_boot_tests();
     load_inst_tests();
     load_reboot_tests();
+    loadtest "kernel/mellanox_config";
+    load_reboot_tests();
```

Comment:
there is a lot of load_reboot_tests() here. Should we avoid this and have it implicitly reboot from mellanox_config, or should we keep it explicitely here? This leads to quite a lot of duplicate lines all over the place..

Reply:
Let me do a little refactor in main.pm moving this functions to load_baremetal_tests and grouping things together to avoid duplicated lines. But I would prefer to do it in a later patch.